src/main.go

```diff
@@ -18,6 +18,8 @@ func main() {
 		jitsiURL   = os.Getenv("BOT_JITSI_URL")
 	)
 
+	log.Printf(`Starting NextcloudTalk-Jitsi-Bot on "%v" (Jitsi: "%v")`, url, jitsiURL)
```

Comment:
Maybe the repo name or "Nextcloud Talk Jitsi Bot" is a better naming scheme here? I know we Germans have a few weird conventions when it comes to _Neologismen_ ;)

Reply:
Just a matter of tast. Besidethe "german" habits: If we do use spaces, the target is only visible in the text if you quote it. CamelCase could be uses, as the do in best-practice in toml-files in rust. :)